Hosted Login: Authentication Profile (#294)
gobeyondidentity/developer-docs: docs/authentication/authentication.mdx

import { Badge } from "@nextui-org/react";

Currently the following authentication factors are supported and more are coming soon:

- Universal Passkey

- FIDO2 Multi-Device Passkey - <Badge color="primary" variant="flat" >Coming soon!</Badge>

> - Email OTP - <Badge color="primary" variant="flat" >Coming soon!</Badge>

Reviewer: We have this now! :)

- Federated IDP - <Badge color="primary" variant="flat" >Coming soon!</Badge>

## Hosted Web

> The Hosted Web Authenticator is a constrained version of the Embedded SDK authenticator type. It requires less configuration and code, but it does require redirecting your users to a web-based authenticator hosted by Beyond Identity to utilize your application's passkeys on a user's device. However, the Hosted Web Authenticator can be configured with your own logo and brand colors. Beyond Identity's hosted web app handles passkey registration and authentication for you, including generating new passkeys, presenting users with authenticator choice options as needed, and validating passkey assertions. Your app simply needs to redirect to Beyond Identity's hosted web authenticator, and we do the rest. This option works well with any OIDC library.

Reviewer: I've been thinking about this one haha. I think this might help direct them to create an application and give some guidance on OIDc without being a separate guide.

```suggestion
The Hosted Web Authenticator is a constrained version of the Embedded SDK authenticator type. It requires less configuration and code, but it does require redirecting your users to a web-based authenticator hosted by Beyond Identity. The web-based authenticator, however, can be configured with your own logo and brand colors. The Hosted Web Authenticator handles passkey registration and authentication for you, including generating new passkeys, presenting users with authenticator choice options as needed, and validating passkey assertions. Your app simply needs to redirect to Beyond Identity's hosted web authenticator, and we do the rest. To use the Hosted Web Authenticator, your application needs an OpenID Connect client using the Authorization Code flow. We recommend looking for one on the list of [certified OpenID Connect clients](https://openid.net/developers/certified/). To get started, [create an application](/docs/next/add-an-application). All the values you need for your OpenID Connect client can be found in your Beyond Identity console application’s External Protocol tab. 
```

import { Badge } from "@nextui-org/react";

Currently the following authentication factors are supported and more are coming soon:

- Universal Passkey

- FIDO2 Multi-Device Passkey - <Badge color="primary" variant="flat" >Coming soon!</Badge>

- Email OTP - <Badge color="primary" variant="flat" >Coming soon!</Badge>

- Federated IDP - <Badge color="primary" variant="flat" >Coming soon!</Badge>

## Hosted Web

> The Hosted Web Authenticator is a constrained version of the Embedded SDK authenticator type. It requires less configuration and code, but it does require redirecting your users to a web-based authenticator hosted by Beyond Identity to utilize your application's passkeys on a user's device. The Hosted Web Authenticator is a Beyond Identity hosted page that embeds the Javascript SDK. Think of this as a wallet hosted by Beyond Identity Cloud that holds an identity's passkeys.

Reviewer: Can we update the copy here. It's also no longer a wallet. 

```suggestion
The Hosted Web Authenticator is a constrained version of the Embedded SDK authenticator type. It requires less configuration and code, but it does require redirecting your users to a web-based authenticator hosted by Beyond Identity to utilize your application's passkeys on a user's device. The Hosted Web Authenticator is a Beyond Identity hosted page that embeds the Javascript SDK.
```

Hosted Web Authenticator is a constrained version of the Embedded Authenticator Config with the following default values:

> <ul><li><b>Invocation Type</b>: automatic </li><li><b>Invoke URL</b>: https://auth-&#123;REGION&#125;.beyondidentity.com/authenticator/</li><li><b>Trusted Origin</b>: [https://auth-&#123;REGION&#125;.beyondidentity.com/authenticator/]</li></ul>

Reviewer: Can we remove this? Invocation Type is correct but the other values are out of date. I'm also not sure it's useful to know this in a low code solution either.

```suggestion

```

:::tip KEEP IN MIND
Passkeys are tied to an origin and they are stored on a specific browser.

> :::

Reviewer: Can we remove this? `Invocation Type` is correct but the other values are out of date. I'm also not sure it's useful to know this in a low code solution either.  The passkey part is out of place too.

> :::tip KEEP IN MIND
> Passkeys are tied to an origin and they are stored on a specific browser.
> :::

Reviewer: The passkey part is out of place too. We can probably remove this. It might be useful somewhere else
```suggestion
```
